### PR TITLE
Update documentation and update links

### PR DESCRIPTION
**Pull request by showierdata9978**

### CODE REVIEW

File: docs/api/features.md
Comment (mybearworld) [inline docs/api/features.md:1]: What's the point of this file? Shouldn't this documentation be for the Meower API, not the features Meower has?

File: docs/api/objects/post.md [hunk 1]
```diff
@@ -2,21 +2,25 @@
...
-| t | [Extended Timestamp](./extended-timestamp) object | The post's creation timestamp. | |
-| u | string | The username of the post author. | |
-| p | string | The content of the post. | |
-| unfiltered_p | string | The non-filtered content of the post. | ✓ |
-| pinned | boolean | Whether the post is pinned. | |
-| isDeleted | boolean | Whether the post is deleted. | |
-| mod_deleted | boolean | Whether the post was deleted by a moderator. | ✓ |
-| deleted_at | integer | The post's deletion timestamp in Unix seconds. | ✓ |
+<!-- deno-fmt-ignore-start --> <!-- Uploads was down -->
+| Field        | Type                                              | Description                                                                              | Optional |
```
Comment (mybearworld) [inline docs/api/objects/post.md:6]: Could this be formatted the way all the other tables are?

**Review by mybearworld — CHANGES_REQUESTED** [review]

File: docs/api/objects/post.md (hunk 1 above)
Comment (showierdata9978) [reply docs/api/objects/post.md:6]: no, it's unreadable and uneditable otherwise

File: docs/api/features.md
Comment (showierdata9978) [reply docs/api/features.md:1]: No other place to put it in the docs. Its could be considered a semi index for where stuff is as it gets documented

Comment (mybearworld) [reply docs/api/features.md:1]: What's the point of it in the first place, though? How does it make the documentation better? People reading the Meower docs probably already know which features Meower has

File: docs/api/objects/post.md (hunk 1 above)
Comment (mybearworld) [reply docs/api/objects/post.md:6]: How? Also, that's what the rest of the tables do, it should really be consistent

File: docs/api/features.md
Comment (showierdata9978) [reply docs/api/features.md:1]: the client requirements where the only place it existed before.  So it needs to be here instead. This is especially needed for new client devs that cant read python/golang

File: docs/api/objects/post.md (hunk 1 above)
Comment (showierdata9978) [reply docs/api/objects/post.md:6]: I had to format it with a vscode ext so I could actually edit it

Comment (showierdata9978) [reply docs/api/objects/post.md:6]: (tried to do it with 1 other but the ext failed to format that one)

File: docs/api/features.md
Comment (mybearworld) [reply docs/api/features.md:1]: > So it needs to be here instead. This is especially needed for new client devs that cant read python/golang

Meower Svelte and others exist. People can see the features Meower has from there. Specific implementation details (i.e. API endpoints) is what the docs would be for

File: docs/api/objects/post.md (hunk 1 above)
Comment (mybearworld) [reply docs/api/objects/post.md:6]: > I had to format it with a vscode ext so I could actually edit it

What do you mean by that? You can edit it when it's formatted without fifty million spaces and dashes

File: docs/api/features.md
Comment (showierdata9978) [reply docs/api/features.md:1]: yes but that still requires reading source code. It's slightly better to have them listed out somewhere

File: docs/api/objects/post.md (hunk 1 above)
Comment (showierdata9978) [reply docs/api/objects/post.md:6]: Its uneditable to me because the formatting is inreadable when compressed

File: docs/api/features.md
Comment (mybearworld) [reply docs/api/features.md:1]: Listing the API endpoints is the entire point of the docs, doesn't need its own file

File: docs/api/objects/post.md (hunk 1 above)
Comment (mybearworld) [reply docs/api/objects/post.md:6]: Compressed?

Comment (showierdata9978) [reply docs/api/objects/post.md:6]: like unformatted

Comment (mybearworld) [reply docs/api/objects/post.md:6]: I really don't see what you mean

_"Formatted"_
image

_Current_
image

Comment (showierdata9978) [reply docs/api/objects/post.md:6]: gh wraps lines

Comment (mybearworld) [reply docs/api/objects/post.md:6]: Yes?

Comment (showierdata9978) [reply docs/api/objects/post.md:6]: Meaning it looks worse with line wrapping because it adds an extra line inbetween lines

Comment (mybearworld) [reply docs/api/objects/post.md:6]: Even without word wrapping (but especially with word wrapping which is pretty much necessary for md files), the current is way easier to read and has more information on screen

_"Formatted"_
image

_"Current"_
image

File: docs/api/features.md
Comment (williamhorning) [inline docs/api/features.md:1]: there's no reason for this, especially if you're not adding it to the sidebar. the api reference (/api/rest) should be sufficient once the openapi schema is updated to have actual descriptions and names for the endpoints

File: docs/api/objects/emoji.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+<!-- deno-fmt-ignore-start -->
+
+| Field | Type | Description |
+| - | - | - |
+| _id | string | The emoji's UUID4. |
+| animated | boolean | Whether the emoji is animated. |
+| chat_id | UUID4 | The chat's UUID4. |
+| name | string | The emoji's name. |
+
+<!-- deno-fmt-ignore-end -->
```
Comment (williamhorning) [inline docs/api/objects/emoji.md:14]:
```suggestion
<!-- deno-fmt-ignore-start -->
| Field | Type | Description |
| - | - | - |
| _id | string | The emoji's UUID4. |
| animated | boolean | Whether the emoji is animated. |
| chat_id | UUID4 | The chat's UUID4. |
| name | string | The emoji's name. |
<!-- deno-fmt-ignore-end -->
```

please be consistent w/formatting the ignore blocks

File: docs/api/objects/post.md
Comment (williamhorning) [inline docs/api/objects/post.md:1]: please change table formatting to match the other existing files and remove the comment about uploads being down

File: docs/api/objects/post.md [hunk 3]
```diff
@@ -25,26 +29,74 @@
...
-	"u": "eri",
-	"p": "It’s gonna be great:tm:",
-	"isDeleted": false
+	"type": 1,
+	"u": "ShowierData9978"
 }
 ```
 
-#### Livechat
+#### Livechat (outdated)
```
Comment (williamhorning) [inline docs/api/objects/post.md:99]: for this and the other instances of things being marked as outdated, could you please actually update the examples

**Review by williamhorning — CHANGES_REQUESTED** [review]

File: docs/api/features.md
Comment (mybearworld) [reply docs/api/features.md:1]: > once the openapi schema is updated to have actual descriptions and names for the endpoints

Is this planned? That would leave the REST-API section pretty much useless

Comment (williamhorning) [reply docs/api/features.md:1]: > Is this planned? That would leave the REST-API section pretty much useless

yes. we'd like to eventually improve the autogenerated documentation, but there're limitations w/what we can have them display which would probably keep the handwritten docs around

Comment (showierdata9978) [reply docs/api/features.md:1]: That was very overwhelming tbf

**Pull request closed without merging**